system-monitor/avg_22_03_2021_10_27_38.csv

```diff
@@ -0,0 +1,97 @@
+function_name,timestamp,PID,CPU,RAM
```

Comment:
This file should be excluded. Moreover, it shouldn't be created in this destination. As far as I understand you set the path to `system-monitor/experiments/...`